Analysis/HTauTauAnalysis/main_HTauTauAnalysis.C

```diff
@@ -0,0 +1,302 @@
+//////////////////////////////////////////////////////////////////////////////////////////
+// Creates a TChain to be used by the Analysis.C class
+#include "TROOT.h"
+#include "TChain.h"
+#include "TFile.h"
+
+void main_HTauTauAnalysis(int proof = 0, int option= 0){
+  
+  /* The URL to the CERN Open Data portal repository */
+  TString path = "/eos/opendata/atlas/rucio/opendata/";
```

Comment:
```suggestion
  TString path = "root://eospublic.cern.ch//eos/opendata/atlas/rucio/opendata/";
```

This should make the code work on any system that has xrootd, instead of just those with eos mounted